vendor/bat-native-ads/src/bat/ads/internal/processors/contextual/text_embedding/text_embedding_processor.cc

```diff
@@ -54,15 +56,17 @@ void TextEmbedding::Process(const std::string& html) {
 
   const std::string text = SanitizeHtml(html);
   if (text.empty()) {
-    BLOG(1, "No text available for embedding");
+    BLOG(1, "Not enough words to embed text");
```

Comment:
This blog comment should remain as `"No text available for embedding"`. This one here is just for sanitizing the html text, rather than checking for in-vocabulary words. It's the lines further below around line ~70 that should blog `"Not enough words to embed text"`, in the case that no in-vocabulary words were found (which we can see from `text_embedding.embedding` being a 0-vector.